src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5019,7 +5019,6 @@ export class StudyViewPageStore
                                     ),
                                 };
                             })
-                            .filter(record => record.count > 0)
```

Comment:
will we now show options even when there are no matching samples?

Reply:
@dippindots this filtering must have a purpose (don't show options what have no samples?)

Reply:
@alisman the way it looks now is the options actually say "0 samples" in the UI. so this just makes the UI still appear even if there are no profiles with profiled, selected samples. the main word there is selected - so there could be profiled samples in the study, but not in the current study view filter. I think this is the most consistent way to handle it since we're not hiding any other add chart UI based on the study view filter state

Reply:
@alisman Agreed with Adam, based on the discussion yesterday during the meeting, I think we decided to show all profiles regardless of it has samples or not, so even if a profile has `0 samples`, we will still show that option.